node/declaration.ts

```diff
@@ -32,17 +34,48 @@ export const getDeclarationsFromConfig = (
   for (const declaration of [...tomlDeclarations, ...deployConfig.declarations]) {
     const config = functionsConfig[declaration.function] ?? {}
 
+    // If no config is found, add the declaration as is
+    if (Object.keys(config).length === 0) {
+      declarations.push(declaration)
+    }
+
+    // log if the path property is not an array
+    if (config.path && !Array.isArray(config.path)) {
+      log.user(
+        `Can't use in source config path property: ${config.path}. Function configuration path property must be an array.`,
+      )
```

Comment:
Not sure how far we have rolled out ISC for edge functions yet, but do we need to provide backwards compatibility? So if path is a string, simply convert it to an array and print a warning?

Reply:
This is now done.